CMakeLists.txt

```diff
@@ -3,44 +3,45 @@ cmake_minimum_required(VERSION 3.0)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_LIST_DIR}/cmake")
 find_package(SDL2 REQUIRED)
-include_directories(${SDL2_INCLUDE_DIR})
+if(WIN32)
+	include_directories("${CMAKE_CURRENT_LIST_DIR}/windows")
```

Comment:
I would put it in another folder.
Maybe in `msvc/include` or `support/msvc/include`.
Maybe @mist64 has a preference

Reply:
I moved it to `support/msvc/include` for now.